i18n/en.json

```diff
@@ -281,6 +281,8 @@
   "admin.cluster.ClusterNameDesc": "The cluster to join by name.  Only nodes with the same cluster name will join together.  This is to support Blue-Green deployments or staging pointing to the same database.",
   "admin.cluster.ClusterNameEx": "E.g.: \"Production\" or \"Staging\"",
   "admin.cluster.enableDescription": "When true, Mattermost will run in High Availability mode. Please see [documentation](!http://docs.mattermost.com/deployment/cluster.html) to learn more about configuring High Availability for Mattermost.",
+  "admin.cluster.EnableExperimentalGossipEncryption": "Enable Experimental Gossip encryption:",
+  "admin.cluster.EnableExperimentalGossipEncryptionDesc": "When true, all communication through the gossip protocol will be encrypted.",
```

Comment:
Would be nice to have PM review on this UI text.

Reply:
@agnivade can you please modify to "Enable Experimental Gossip Encryption."  

Also wondering if there is a particular encryption method being used that would be valuable for the system admin to know?

Reply:
@wiersgallak - From what I understand, the ":" is there for all other settings though. See ClusterName, GossipPort, StreamingPort, EnableTitle etc. But happy to change it if you recommend.

The encryption method is AES-256 and is done in consultation with the security team. The choices are between AES-128, 192 and 256. The customer can choose to bring down the key size if they choose to. I will add all of these details in the release notes. For a simple config description, going to such details might not be worth it.

Reply:
@agnivade Good catch on the punctuation - my oversight. Just to confirm that there are no other settings to configure the key size? 
Will likely need your help on adding the details to the documentation for this if its something they do outside of the configuration settings in-product.

Reply:
Yes there are no other settings to configure key size. This is by design to make encryption automatic with the flip of a switch.

Yep, I have marked the server PR as "docs needed" and will work with Amy to provide the release docs when 5.26 gets released.